### PR TITLE
CLDR-15271 hi-Latn values

### PR DESCRIPTION
**Pull request by AEApple**
CLDR-15271 

hi-Latn values

- [ ] This PR completes the ticket.

ALLOW_MULTI_COMMITS=true

### CODE REVIEW

File: seed/main/hi_Latn.xml [hunk 1]
```diff
@@ -1006,15 +2104,2224 @@ annotations.
...
+			<metazone type="Atlantic">
+				<long>	
+					<generic>Western Argentina Time</generic>
+					<generic>Atlantic Time</generic>
+					<standard>Western Argentina Standard Time</standard>
+					<standard>Atlantic Standard Time</standard>
+					<daylight>Western Argentina Summer Time</daylight>
+					<daylight>Atlantic Daylight Time</daylight>
+				</long>
+			</metazone>
```
Comment (srl295) [inline seed/main/hi_Latn.xml]: looks a little odd

Comment (srl295) [inline seed/main/hi_Latn.xml]:
```suggestion
			<metazone type="Atlantic">
				<long>	
					<generic>Atlantic Time</generic>
					<standard>Atlantic Standard Time</standard>
					<daylight>Atlantic Daylight Time</daylight>
				</long>
			</metazone>		
```

This is the actual error. But the other duplications are problematic also.

Comment (AEApple) [reply seed/main/hi_Latn.xml]: Thanks!! I have switched to inheritance.

Comment (AEApple) [reply seed/main/hi_Latn.xml]: Fixed! Thank you!

**Comment by srl295** [comment]
@AEApple more errs, see https://github.com/unicode-org/cldr/actions/runs/1748071713

**Comment by srl295** [comment]
checkhi_Latn: display collision @ Timezones\|Overrides\|Pacific/Honolulu\|generic-short
--
checkhi_Latn: display collision @ Timezones\|Overrides\|Pacific/Honolulu\|standard-short
checkhi_Latn: display collision @ Timezones\|Overrides\|Pacific/Honolulu\|daylight-short

**Comment by AEApple** [comment]
> ## checkhi_Latn: display collision @ Timezones|Overrides|Pacific/Honolulu|generic-short
> checkhi_Latn: display collision @ Timezones|Overrides|Pacific/Honolulu|standard-short checkhi_Latn: display collision @ Timezones|Overrides|Pacific/Honolulu|daylight-short

Yes, I just saw the failure notification :)

I want to finish fixing all the errors before adding the remaining data.

**Review by srl295 — DISMISSED** [review]

**Review by srl295 — APPROVED** [review]

**Review by srl295 — APPROVED** [review]

**Pull request merged**